ui-react/packages/atlasmap-ui/src/CanvasView/components/FieldsBoxHeader.tsx

```diff
@@ -82,44 +83,52 @@ export const FieldsBoxHeader: FunctionComponent<IFieldsBoxHeaderProps> = ({
               position={'right'}
               dropdownItems={[
                 <DropdownItem variant={'icon'} key={'import'}>
-                  <DropdownItemIcon>
-                    <ImportIcon />
-                  </DropdownItemIcon>
-                  <Tooltip
-                    position={'auto'}
-                    enableFlip={true}
-                    content={<div>Import instance or schema file</div>}
-                  >
+                  <SplitItem isFilled>
```

Comment:
Is this split item doing anything?

Reply:
Yes, it makes the entire row selectable from the dropdown. Before, clicking the white space behind the menu item label, like 'Import', did nothing.